static/js/components/tiles/highlight_tile.tsx

```diff
@@ -70,6 +70,7 @@ export function HighlightTile(props: HighlightTilePropType): JSX.Element {
             false,
```

Comment:
While here should we make a bit unit smaller?  USD all caps looks a bit odd.

Reply:
This is a bit involved. Need to make formatNumber() function to return 2 parts; maybe a later change.